package.json

```diff
@@ -111,10 +108,7 @@
     "test": "jest",
     "tdd": "jest --watch",
     "updateSnapshot": "jest --updateSnapshot",
-    "cy:open": "cypress open",
-    "cy:run": "cypress run",
-    "test:e2e": "start-server-and-test 'yarn develop' http://localhost:8000 'yarn cy:open'",
-    "test:e2e:ci": "start-server-and-test 'yarn serve -p 8000' http://localhost:8000 'yarn cy:run'",
+    "test:e2e": "start-server-and-test 'yarn serve -p 8888' http://localhost:8888/casei 'npx playwright test -c ./playwright/e2e'",
```

Comment:
@willemarcel should this include the build command?

Reply:
As the build takes a couple of minutes, I prefer to execute it separately. Every so often, we don't need a new build, but just to fix a test file and run the test again.